shanoir-ng-front/src/app/study-cards/study-card/study-card.component.ts

```diff
@@ -144,7 +144,11 @@ export class StudyCardComponent extends EntityComponent<StudyCard> {
 
     private fetchNiftiConverters() {
         this.niftiConverterService.getAll()
-            .then(converters => this.niftiConverters = converters.map(converter => new IdName(converter.id, converter.name)));
+            .then(converters => {
+                this.niftiConverters = converters.map(converter => new IdName(converter.id, converter.name));
+                // pre-select dcm2niix
+                this.studyCard.niftiConverter = converters.filter(element => element.name === 'dcm2niix')[0];
```

Comment:
fails in edit mode, this.studyCard is undefined at this point. Anyway, you don't want to change the converter in edit mode, your dev is only for the create mode. So you must move that code to the initCreate() function :
* Just move one row up this.studyCard = new StudyCard(); (above this.fetchNiftiConverters();)
*  Add a .then to this.fetchNiftiConverters(); : 
    * this.fetchNiftiConverters().then(() => {   put your code here   });